spec/requests/v1/pacbio/requests_spec.rb

```diff
@@ -77,8 +77,8 @@
       context 'filters - source_identifier' do
         it 'when the source_identifier belongs to a plate' do
           pacbio_plate = create(:plate_with_wells_and_requests, pipeline: 'pacbio')
-          pacbio_plate_requests = pacbio_plate.wells.flat_map(&:pacbio_requests)
-          get "#{v1_pacbio_requests_path}?filter[source_identifier]=#{pacbio_plate.barcode}",
+          pacbio_plate_requests = pacbio_plate.wells.first.pacbio_requests
```

Comment:
I don't think this test should need to be changed, indicates a change in behaviour for the filter.